tests/ocall/host/host.cpp

```diff
@@ -77,7 +77,11 @@ int main(int argc, const char* argv[])
     /* Call enc_set_tsd */
     {
         int ret_val = -1;
+#ifdef _WIN32
+        result = enc_set_tsd(enclave, &ret_val, _strdup("TSD-DATA"));
```

Comment:
Same comment about this ifdef as for getpid, since this case is the same, where MSDN explains:
"The Microsoft-implemented POSIX function names strdup and wcsdup are deprecated aliases for the _strdup and _wcsdup functions. By default, they generate Compiler warning (level 3) C4996. The names are deprecated because they don't follow the Standard C rules for implementation-specific names. However, the functions are still supported."